docs/dev/table/sourceSinks.md

```diff
@@ -264,6 +288,30 @@ val source: TableSource[_] = Kafka010AvroTableSource.builder()
 </div>
 </div>
 
+* **Specify the start reading position:** By default, the table source will start reading data from the committed group offsets in Zookeeper or Kafka brokers. You can specify other start positions via the builder's methods, which correspond to the configurations in section [Kafka Consumers Start Position Configuration](../connectors/kafka.html#kafka-consumers-start-position-configuration).
+
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+TableSource source = Kafka010JsonTableSource.builder()
```

Comment:
This should be Kafka010AvroTableSource.

Reply:
Oops, sorry about that and thanks for pointing it out.